### ISSUE TITLE
is the client available as maven artefact in some repo?

### ISSUE THREAD

**Issue by m1ci** [LinkedDataFragments/Client.Java#8]
Hi,
Just wondering whether the java client is available as maven artifact in some online repo?

Thanks!

**Comment by RubenVerborgh** [comment]
Not yet. Any recommendations to do so?

**Comment by m1ci** [comment]
yup, it can be nice to have the client in some maven repo. E.g. http://maven.apache.org/repository/

**Comment by RubenVerborgh** [comment]
Alright, will consider!

**Comment by agazzarini** [comment]
@RubenVerborgh with just few lines we could provide this artifact in a custom maven repository [1](not in the official one). The repository can be hosted everywhere, better if it is in this github repository (as a dedicated branch).  

I already tried for a project of mine, and it's perfectly work. If you want I can submit a patch for that.

AG 

---

[1] http://stackoverflow.com/questions/14013644/hosting-a-maven-repository-on-github

**Comment by RubenVerborgh** [comment]
That would be great!
